cli/pkg/kctrl/cmd/core/semver_value.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 20210 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package core
+
+import (
+	uitable "github.com/cppforlife/go-cli-ui/ui/table"
+	versions "github.com/vmware-tanzu/carvel-vendir/pkg/vendir/versions"
+)
+
+type ValueSemver struct {
+	V uitable.Value
+}
+
+func NewValueSemver(v uitable.Value) ValueSemver {
+	return ValueSemver{V: v}
+}
+
+func (t ValueSemver) String() string {
+	return t.V.String()
+}
+
+func (t ValueSemver) Value() uitable.Value { return t }
+func (t ValueSemver) Compare(other uitable.Value) int {
```

Comment:
should cast other similar to https://github.com/cppforlife/go-cli-ui/blob/master/ui/table/values.go#L80

Reply:
Done.